apps/judicial-system/backend/src/app/modules/defendant/test/limitedAccessDefendantController/updateDefendant.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { uuid } from 'uuidv4'
+
+import { MessageService } from '@island.is/judicial-system/message'
+import {
+  CaseType,
+  PunishmentType,
+  User,
+} from '@island.is/judicial-system/types'
+
+import { createTestingDefendantModule } from '../createTestingDefendantModule'
+
+import { Case } from '../../../case'
+import { UpdateDefendantDto } from '../../dto/updateDefendant.dto'
+import { Defendant } from '../../models/defendant.model'
+
+interface Then {
+  result: Defendant
+  error: Error
+}
+
+type GivenWhenThen = (
+  defendantUpdate: UpdateDefendantDto,
+  type: CaseType,
+  courtCaseNumber?: string,
+) => Promise<Then>
+
+describe('LimitedAccessDefendantController - UpdateDefendant', () => {
+  const user = { id: uuid() } as User
+  const caseId = uuid()
+  const defendantId = uuid()
+  const defendant = {
+    id: defendantId,
+    caseId,
+    nationalId: uuid(),
+    defenderEmail: uuid(),
+  } as Defendant
+
+  let mockMessageService: MessageService
+  let mockDefendantModel: typeof Defendant
+  let givenWhenThen: GivenWhenThen
+
+  beforeEach(async () => {
+    const { messageService, defendantModel, limitedAccessDefendantController } =
+      await createTestingDefendantModule()
+
+    mockMessageService = messageService
+    mockDefendantModel = defendantModel
+
+    const mockUpdate = mockDefendantModel.update as jest.Mock
+    mockUpdate.mockRejectedValue(new Error('Some error'))
+
+    givenWhenThen = async (
+      defendantUpdate: UpdateDefendantDto,
+      type: CaseType,
+      courtCaseNumber?: string,
+    ) => {
+      const then = {} as Then
+
+      await limitedAccessDefendantController
+        .updateDefendant(
+          caseId,
+          defendantId,
+          user,
+          { id: caseId, courtCaseNumber, type } as Case,
```

Comment:
Tiny nitpick, since you don't have any tests right now that have a courtCaseNumber set you could just leave that out of the arguments.